src/.nuget/Microsoft.NETCore.Jit/Microsoft.NETCore.Jit.pkgproj

```diff
@@ -6,15 +6,6 @@
     <IsLineupPackage Condition="'$(IsLineupPackage)' == ''">true</IsLineupPackage>
   </PropertyGroup>
 
-  <PropertyGroup>
-    <LongNamePlatform>$(PackagePlatform)</LongNamePlatform>
-    <LongNamePlatform Condition="'$(LongNamePlatform)'=='x64'">amd64</LongNamePlatform>
-    <CrossTargetPlatform>$(CrossTargetComponentFolder)</CrossTargetPlatform>
-    <CrossTargetPlatform Condition="'$(CrossTargetPlatform)'=='x64'">amd64</CrossTargetPlatform>
-    <LongNameSuffix>_$(LongNamePlatform)_$(LongNamePlatform)_$(MajorVersion).$(MinorVersion).$(BuildNumberMajor).$(BuildNumberMinor)</LongNameSuffix>
```

Comment:
I'd like to understand why these properties are being removed. Did we ever use them for the jit in the first place?

Reply:
They are not used right now. I'm not sure about in the past - they were added to the jit pkgproj as part of a large refactor over two years ago, but even in that refactor I didn't see them used for the jit package.